src/styles/podcast_episode.scss

```diff
@@ -0,0 +1,40 @@
+/**
+Used to style a single podcast episode.
+*/
+
+
+#slot {
```

Comment:
would be nice to have a name that explains where it is used. e.g. episode or something like this?

Reply:
Fixed in https://github.com/EngineeringKiosk/webpage/pull/38/commits/4b7fa200073859cdc3659f924286a9b489e6f13c